CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ pathology_metastasis_detection
 
 ### Model storage
 
-Github limits the size of files allowed in the repository (see [About size limits on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)). Therefore, MONAI Model Zoo limits each single file to be no larger than **25MB**.
+Github limits the size of files allowed in the repository (see [About size limits on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github). Therefore, MONAI Model Zoo limits each single file to be no larger than **25MB**.
```

Comment:
I think this ")" is to match the "(" in "... allowed in the repository (see ...".

Thanks.

Reply:
Yes, you are correct. I've reverted these changes.